charts/cht-chart-4x/templates/upgrade-service-deployment.yml

```diff
@@ -22,7 +22,7 @@ spec:
       restartPolicy: Always
       serviceAccountName: cht-upgrade-service-user
       containers:
-      - image: medicmobile/upgrade-service:0.31
+      - image: medicmobile/upgrade-service:0.32
```

Comment:
Shouls this be broken into an variable?
```suggestion
      - image: medicmobile/upgrade-service:{{ .Values.upgrade-service.image.tag }}
```
and in `values.yaml`:
```yaml
# charts/cht-chart-4x/values.yaml
upgrade-service:
  image:
    tag: "0.32"
```

It being set in the charts values means it has that as the default value but it can be overridden by the user